app/models/admin_auction_status_presenter_factory.rb

```diff
@@ -7,18 +7,16 @@ def initialize(auction:)
 
   def create
     if auction.payment_confirmed? || auction.c2_paid?
-      Object.const_get("C2StatusPresenter::#{c2_status}").new(auction: auction)
+      Object.const_get("C2StatusPresenter::#{c2_status}")
     elsif future? && auction.published?
-      AdminAuctionStatusPresenter::Future.new(auction: auction)
+      AdminAuctionStatusPresenter::Future
     elsif work_in_progress?
-      AdminAuctionStatusPresenter::WorkInProgress.new(auction: auction)
-    elsif auction.accepted? && auction.accepted_at.nil?
-      AdminAuctionStatusPresenter::PendingPaymentUrl.new(auction: auction)
+      AdminAuctionStatusPresenter::WorkInProgress
     elsif !auction.pending_delivery?
-      Object.const_get("AdminAuctionStatusPresenter::#{status}").new(auction: auction)
+      Object.const_get("AdminAuctionStatusPresenter::#{status}")
     else # if auction.purchase_card == 'default'
-      Object.const_get("C2StatusPresenter::#{c2_status}").new(auction: auction)
-    end
+      Object.const_get("C2StatusPresenter::#{c2_status}")
+    end.new(auction: auction)
```

Comment:
Having a common interface to all presenters is nice, which is why we have the `Base` classes. As of right now, they can all be initialized the same way. If we get to a point where they are very different, that might be the code telling us the we need separate factories